### PR TITLE
Kotlin 1.4

### PR DESCRIPTION
**Pull request by asarazan**
It builds, at least. Haven't gotten around to testing yet. Which is fortunate because the coroutine test library is borkened

### CODE REVIEW

File: app/src/commonMain/kotlin/net/sarazan/bismarck/mobile/Api.kt [hunk 1]
```diff
@@ -14,12 +14,12 @@ import io.ktor.http.ContentType
...
 import io.ktor.http.takeFrom
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.nonstrict
 
 class Api {
-    @OptIn(kotlinx.serialization.UnstableDefault::class)
     private val client = HttpClient {
         install(JsonFeature) {
-            serializer = KotlinxSerializer(Json.nonstrict)
+            serializer = KotlinxSerializer(Json { isLenient = true })
```
Comment (treelzebub) [inline app/src/commonMain/kotlin/net/sarazan/bismarck/mobile/Api.kt:22]: this made me lol

File: app/src/commonMain/kotlin/net/sarazan/bismarck/mobile/Feed.kt [hunk 2]
```diff
@@ -75,8 +77,9 @@ enum class MIMEType(val value: String) {
     AudioMPEG("audio/mpeg");
 
     companion object : KSerializer<MIMEType> {
-        override val descriptor: SerialDescriptor get() {
-            return PrimitiveDescriptor("quicktype.MIMEType", PrimitiveKind.STRING)
+        override val descriptor: SerialDescriptor
+            get() {
+            return PrimitiveSerialDescriptor("quicktype.MIMEType", PrimitiveKind.STRING)
```
Comment (treelzebub) [inline app/src/commonMain/kotlin/net/sarazan/bismarck/mobile/Feed.kt:82]: indentarooni

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,21 +1,6 @@
...
-# limitations under the License.
-#
-
+#Mon Aug 31 17:45:31 PDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```
Comment (treelzebub) [inline gradle/wrapper/gradle-wrapper.properties:6]: orly

**Comment by asarazan** [comment]
All tests pass, publish to mavenLocal succeeds, can be consumed by other project on same machine. Gonna do a final lint+dependency pass later and then publish

**Comment by asarazan** [comment]
Published 0.3.0 to mavenCentral. Merging PR!

**Comment by treelzebub** [comment]
👏🏻👏🏻👏🏻👏🏻🎉🎉🎉🎉

**Pull request merged**